### PR TITLE
TILA-396 | organization active members

### PR DESCRIPTION
**Pull request by nicobav**
Adds active members to organisation and to API (nested in application)

### CODE REVIEW

**Review by kieferbonk — APPROVED** [review]

File: api/applications_api.py [hunk 1]
```diff
@@ -67,6 +73,9 @@ class Meta:
             "year_established": {
                 "help_text": "Year when the organisation was established.",
             },
+            "active_members": {
+                "help_text": "Active space user person count of the organization.",
```
Comment (aprenfors) [inline api/applications_api.py:77]: Active space user person count?

Comment (nicobav) [reply api/applications_api.py:77]: :laughing:  yes
I tried to point out perhaps the obvious and surely sounds like a "rabbit hole". 
Perhaps just  "Number of active persons in the organization"?

Comment (aprenfors) [reply api/applications_api.py:77]: Yeah, that sounds good :)

Comment (nicobav) [reply api/applications_api.py]: Fix'd

**Review by aprenfors — APPROVED** [review]

**Pull request merged**